README.md

```diff
@@ -201,13 +201,29 @@ Currently, there is one native companion app available to connect to Tor nodes:
 
 We welcome and appreciate new contributions.
 
-We use the [Development Project Board](https://github.com/getAlby/lightning-browser-extension/projects/2) to plan to-dos. Best choose something from the to-do column. (If there is nothing for you, feel free to pick something from the backlog)
+### Find a task
 
-If you're a developer looking to help but not sure where to begin, check out the issues that have specifically been marked as being friendly to new contributors [here](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+project%3AgetAlby%2Flightning-browser-extension%2F2).
+We use the [Development Project Board](https://github.com/orgs/getAlby/projects/10/views/2) to plan to-dos. Best choose something from the to-do column. (If there is nothing for you, feel free to pick something from the backlog)
 
-> Alby enforces [Conventional Commits Specification](https://www.conventionalcommits.org/en/) for adding human and machine readable meaning to commit messages, make sure to read and follow them.
+If you're a developer looking to help but not sure where to begin, check out the issues that have specifically been [marked as being friendly to new contributors](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+project%3AgetAlby%2Flightning-browser-extension%2F2).
```

Comment:
should we only link to the "good first use" issues? 
https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

I guess the filter with the project board limits it too much?

and what can we give non developers?

If you're interested in the UI and design of Alby, check out the [issues marked as design](https://github.com/getAlby/lightning-browser-extension/issues?q=is%3Aopen+is%3Aissue+label%3A%22design%22+) 

what else can we add? documentation/guides?

Reply:
Good point! Extended the section. Have a look.  
Also had a look at other projects, i.e. https://github.com/Microsoft/vscode#contributing  
I think there's space to split things up in separate doc-files or wiki-pages but we can do that when there's a need and this Readme is getting i.e. too big.
What do you think?